crates/mock/src/utils.rs

```diff
@@ -0,0 +1,182 @@
+use base64::engine::general_purpose;
+use base64::Engine;
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::path::Path;
+use std::str::FromStr;
+
+use serde_json::{self};
+
+use solana_client::rpc_client::RpcClient;
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::{account::Account, genesis_config::ClusterType};
+use yellowstone_grpc_proto::geyser::{SubscribeUpdateAccount, SubscribeUpdateAccountInfo};
+use yellowstone_vixen_core::Parser;
+use yellowstone_vixen_parser::{TokenProgramParser, TokenProgramState};
+
+use crate::async_client::fetch_account_async;
+
+const FIXTURES_PATH: &str = "./fixtures";
```

Comment:
Another spot to be configurable @ray-kast

Reply:
i feel we can add it to .env file itself wyt?